script.js

```diff
@@ -61,130 +63,109 @@ app = {
   }
 }
 
+var margin = {
+    left: 75,
+    right: 50,
+    top: 50,
+    bottom: 75
+};
+
 function Chart(selector) {
-  var chart = this;
 
-  // SVG and MARGINS
+  var chart = this;
 
   var margin = {
-    top: 15, right: 15, bottom: 40, left: 45
+    left: 75,
+    right: 50,
+    top: 50,
+    bottom: 75
   };
 
-  chart.width = 600 - margin.left - margin.right;
-  chart.height = 400 - margin.top - margin.bottom;
 
-  chart.svg = d3.select(selector)
+chart.width = 800 - margin.left - margin.right;
+chart.height = 500 - margin.top - margin.bottom;
+
+chart.svg = d3.select(selector)
     .append('svg')
     .attr('width', chart.width + margin.left + margin.right)
     .attr('height', chart.height + margin.top + margin.bottom)
     .append('g')
     .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
+    
+//Scales - Hopefully 
 
-  // SCALES
+// SCALES
 
   chart.x = d3.scaleLinear()
-    .domain([0, d3.max(app.data, function (d) { return d.total_fertility; })])
+    .domain([d3.min(app.data, function (d) { return d.lon; }),d3.max(app.data, function (d) { return d.lon; })])
     .range([0, chart.width])
     .nice();
```

Comment:
.nice() probably isn't needed here, because you're never going to show an axis that you'd like to end on a nice round number.